lib/membrane/video_compositor/implementation/ffmpeg.ex

```diff
@@ -1,11 +1,11 @@
-defmodule Membrane.VideoCompositor.FFmpeg do
+defmodule Membrane.VideoCompositor.Implementation.FFmpeg do
```

Comment:
```suggestion
defmodule Membrane.VideoCompositor.Implementations.FFmpeg do
```

We usually use the plural for such things, as singular is useful for things like `Implementation.Utils`. But do as you prefer